languages/ruby/lib/oso/polar/polar.rb

```diff
@@ -34,9 +34,10 @@ class Polar # rubocop:disable Metrics/ClassLength
       # @return [Host]
       attr_reader :host
 
-      def initialize
+      def initialize # rubocop:disable Metrics/MethodLength
```

Comment:
lol rubocop such a pain

Reply:
Yeah the "method complexity" one was particularly offensive to me